ingestion/cadet.yaml

```diff
@@ -10,6 +10,11 @@ source:
     target_platform: athena
     target_platform_instance: athena_cadet
     infer_dbt_schemas: true
+    node_name_pattern:
+      deny:
+        - ".*use_of_force\\.summary_status_complete_dim.*"
```

Comment:
what's the significance of ignoring these specific ones? Is it worth adding a comment?